google/cloud/logging_v2/handlers/structured_log.py

```diff
@@ -36,6 +36,24 @@
     "}"
 )
 
+GCP_STRUCTURED_LOGGING_FIELDS = frozenset(
```

Comment:
nit: is that list described anywhere in documentation? I wonder if we could provide any links, so in future the list can be tracked